2.7/s2i/bin/assemble

```diff
@@ -10,7 +10,8 @@ function should_collectstatic() {
 
 function virtualenv_bin() {
   if head "/etc/redhat-release" | grep -q "^CentOS Linux release 7" || \
-     head "/etc/redhat-release" | grep -q "^Red Hat Enterprise Linux\( Server\)\? release 7"; then
+     head "/etc/redhat-release" | grep -q "^Red Hat Enterprise Linux\( Server\)\? release 7" || \
+     head "/etc/redhat-release" | grep -q "^Fedora release 29 \(Twenty Nine\)\?"; then
```

Comment:
`virtualenv-3.7` doesn't work on F29? If we it does, I'd prefer to use it: to be explicit, and to keep uniformity with F28.

Reply:
And if it doesn't, could we do it the other way and use unversioned `virtualenv` also on F28? Or is that still python2?

Reply:
Unless I am missing something, this is the only way how to do it because:

* `virtualenv` is provided by python2-virtualenv on F28.
```
# rpm -qf /usr/bin/virtualenv
python2-virtualenv-16.0.0-4.fc28.noarch
```
* and python3-virtualenv provides only `virtualenv` without any suffix on F29.
```
# rpm -ql python3-virtualenv | grep bin
/usr/bin/virtualenv
```

Reply:
Damn, that's pretty horrible... but what can we do. Ok, LGTM.